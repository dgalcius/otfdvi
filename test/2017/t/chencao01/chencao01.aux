\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\zref@newlabel[2]{}
\catcode `"\active 
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand*\HyPL@Entry[1]{}
\citation{1.Couzin2008,1.Couzin2005,1.Czirok1999,1.Spieser2009,1.Moussaid2011}
\citation{1.Cristiani2011,1.Bellomo2012,1.DirkHelbing2005,1.Bellomo2011b,1.Dogbe2012a}
\HyPL@Entry{0<</S/D>>}
\vktinfo{kt-ltx-chk}{cmn}{cmn_dg_all}
\vktinfo{kt-ltx-chk}{cmn}{cmn_dg_front}
\vktinfo{kt-ltx-chk}{cmn}{cmn_dg_body}
\vktinfo{kt-ltx-chk}{cmn}{cmn_vln_all}
\vktinfo{kt-ltx-chk}{cmn}{cmn_vln_ext}
\vktinfo{kt-ltx-chk}{ltx}{ltx_dg_body}
\vktinfo{kt-ltx-chk}{xml}{xml_dg_all}
\vktinfo{kt-ltx-chk}{ltx}{ltx_dg_biblio}
\vktinfo{kt-ltx-chk}{cmn}{cmn_dg_preamble}
\gdef\make@mdfive@files{chencao01.aux,chencao01.out,chencao01.idx,chencao01.pos}
\select@language{english}
\@writefile{toc}{\select@language{english}}
\@writefile{lof}{\select@language{english}}
\@writefile{lot}{\select@language{english}}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}Introduction}{1}{chencao01.chapter.1:1}}
\@writefile{lof}{\addvspace {13\p@ }}
\@writefile{lot}{\addvspace {13\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {1.1}Motivation}{1}{chencao01.section.1:2}}
\@writefile{toc}{\contentsline {section}{\numberline {1.2}Current status of research}{1}{chencao01.section.2:3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.2.1}Modeling of crowds}{1}{chencao01.subsection.1:4}}
\citation{1.Berlonghi1995}
\citation{1.DirkHelbing2005,1.Bellomo2011a,1.Cristiani2011,1.ChristianDogbe2012,1.Bogdan2011a,1.DanielStuart2013}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.2.2}Control of crowds}{2}{chencao01.subsection.2:5}}
\citation{1.AlfioQuarteroni2003,1.Haken2006,1.Thalmann2007,1.WolframW.F2008,1.Bellomo2008a,1.Kachroo2008b,1.Pelechano2008,1.Timmermans2009,1.Kachroo2009,1.Barcelo2010}
\citation{1.BruceJ.West2014}
\citation{1.Ishiwata2012}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.2.3}Comments}{3}{chencao01.subsection.3:6}}
\@writefile{default}{\indexentry{Calculus of fractional order|hyperpage}{3}}
\@writefile{default}{\indexentry{Calculus of fractional order|hyperpage}{3}}
\bibcite{1.Bellomo2008a}{{1}{}{{}}{{}}}
\bibcite{1.Bellomo2011b}{{2}{}{{}}{{}}}
\bibcite{1.Bellomo2012}{{3}{}{{}}{{}}}
\bibcite{1.Bellomo2011a}{{4}{}{{}}{{}}}
\bibcite{1.Berlonghi1995}{{5}{}{{}}{{}}}
\bibcite{1.Bogdan2011a}{{6}{}{{}}{{}}}
\@writefile{lof}{\contentsline {figure}{\numberline {Figure 1.1}{\ignorespaces Organization of this book.\relax }}{4}{chencao01.figure.caption.8}}
\@cons\caption@pkg@list{{ragged2e}}
\providecommand*\caption@xref[2]{\@setref\relax\@undefined{#1}}
\newlabel{1.Fig:organization}{{1.1}{4}{Organization of this book.\relax }{chencao01.figure.caption.8}{}}
\@writefile{toc}{\contentsline {section}{\numberline {1.3}Organization of this book}{4}{chencao01.section.3:7}}
\@writefile{toc}{\contentsline {starsection}{\numberline {}References}{4}{chencao01.section.3:9}}
\bibcite{1.Dogbe2012a}{{7}{}{{}}{{}}}
\bibcite{1.Couzin2008}{{8}{}{{}}{{}}}
\bibcite{1.Couzin2005}{{9}{}{{}}{{}}}
\bibcite{1.Cristiani2011}{{10}{}{{}}{{}}}
\bibcite{1.Czirok1999}{{11}{}{{}}{{}}}
\bibcite{1.ChristianDogbe2012}{{12}{}{{}}{{}}}
\bibcite{1.Haken2006}{{13}{}{{}}{{}}}
\bibcite{1.DirkHelbing2005}{{14}{}{{}}{{}}}
\bibcite{1.Ishiwata2012}{{15}{}{{}}{{}}}
\bibcite{1.Barcelo2010}{{16}{}{{}}{{}}}
\bibcite{1.Kachroo2009}{{17}{}{{}}{{}}}
\bibcite{1.Kachroo2008b}{{18}{}{{}}{{}}}
\bibcite{1.WolframW.F2008}{{19}{}{{}}{{}}}
\bibcite{1.Moussaid2011}{{20}{}{{}}{{}}}
\bibcite{1.Pelechano2008}{{21}{}{{}}{{}}}
\bibcite{1.AlfioQuarteroni2003}{{22}{}{{}}{{}}}
\bibcite{1.Spieser2009}{{23}{}{{}}{{}}}
\bibcite{1.DanielStuart2013}{{24}{}{{}}{{}}}
\bibcite{1.Thalmann2007}{{25}{}{{}}{{}}}
\bibcite{1.Timmermans2009}{{26}{}{{}}{{}}}
\bibcite{1.BruceJ.West2014}{{27}{}{{}}{{}}}
\providecommand\NAT@force@numbers{}\NAT@force@numbers
\gdef\thebibliography@maxwlabel{20}
\newlabel{LastPage}{{}{5}{}{page.5}{}}
\xdef\lastpage@lastpage{5}
\xdef\lastpage@lastpageHy{5}
