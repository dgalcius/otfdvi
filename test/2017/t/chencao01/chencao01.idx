\indexentry {Calculus of fractional order|hyperpage}{3}
\indexentry {Calculus of fractional order|hyperpage}{3}
