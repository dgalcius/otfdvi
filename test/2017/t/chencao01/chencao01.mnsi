<secfile name="Manuscript info">
  <operation>Submit</operation>
  <sec name="CHENCAO01">
    <key name="Main file type">PDF</key>
    <key name="Main file name">print/CHENCAO01.pdf</key>
    <key name="Stage">300</key>
    <key name="LaTeX stage">\CRC</key>
    <key name="Publication year">2018</key>
    <key name="First logical page" type="integer">1</key>
    <key name="Last logical page" type="integer">5</key>
    <key name="Page numbering">arabic</key>
    <key name="Page count" type="integer">5</key>
    <sec name="DB info">
      <sec name="PTS Read">
        <key name="Host">db.vtex.vmt</key>
        <key name="Port">3306</key>
        <key name="User">maket</key>
        <key name="Password">D53B58</key>
        <key name="Database">ptsIII</key>
        <key name="Timeout" type="integer">30</key>
      </sec>
      <sec name="PTS Client">
        <key name="Host">www.vtex.vmt</key>
        <key name="Port" type="integer">80</key>
        <key name="End point">/db/xmlrpc/s1.php</key>
        <key name="Post method">ptspost</key>
        <key name="Test method">help</key>
        <key name="Backup directory">y:\db_backup</key>
        <key name="SSL enabled" type="boolean">No</key>
      </sec>
      <sec name="PTS Info">
        <key name="PTS Info Address">http://pts.vtex.vmt/db/vtexpts/item/check/oper</key>
        <key name="VTEXPTS MS info URL">http://pts.vtex.vmt/db/vtexpts/api/item/%s/info</key>
        <key name="Elsevier journal stylesheets address"></key>
        <key name="PTSIII info URL"></key>
        <key name="Project info URL"></key>
        <key name="VTEXPTS worker info URL">https://pts.vtex.vmt/db/vtexpts/worker/info?uid=%s</key>
      </sec>
    </sec>
    <sec name="File info">
      <key name="Publisher">DGBOOK</key>
      <key name="Project">CHENCAO</key>
      <key name="Manuscript">CHENCAO01</key>
      <key name="Stage">308</key>
      <key name="MS position"></key>
      <key name="Computer name">ECRC594</key>
      <key name="User name">ispudulyte</key>
      <key name="TID">ispudulyte</key>
      <key name="Pages"></key>
      <key name="Format">2016</key>
      <key name="Distribution">vtex</key>
      <key name="PDF type">print</key>
      <key name="LaTeX file name">chencao01.tex</key>
      <key name="Destination">PDF</key>
      <key name="ISSN/ISBN">978-3-11-047281-3</key>
      <key name="Opened from">server</key>
      <key name="PDF.Maker">pdfwf_luatex</key>
      <key name="Spelled">Dictionary: American, Computer: 1GSRED478, 2018.02.22 08:31</key>
      <key name="Graphics directory"></key>
      <key name="Copy to DB" type="boolean">Yes</key>
      <key name="Compiler cmd line"></key>
      <key name="VTEXPTS stage">E200</key>
      <sec name="History" type="text">
        2018.02.21 12:37
      </sec>
    </sec>
    <sec name="LaTeX info">
      <key name="Publisher">DGBook</key>
      <key name="Project">CHENCAO</key>
      <key name="Version"></key>
      <key name="DocType"></key>
      <key name="Format">2016</key>
      <key name="Distribution">vtex</key>
      <key name="Destination">PDF</key>
      <key name="DVI.Maker">luadvi</key>
      <key name="DVI.Viewer">xdviwindo</key>
      <key name="PS.Maker">xdvipsk</key>
      <key name="PS.Viewer">GSView</key>
      <key name="PDF.Maker">pdfwf_luatex</key>
      <key name="PDF.Viewer">Acrobat</key>
      <key name="Run before Open">fhgextract,pdfwf_images (full),VTEXPTS2LaTeX,book_aux_toc,Open Source PDF allways</key>
      <key name="Run before Make"></key>
      <key name="Run after Make">FlsUpdated</key>
      <key name="Run before Send">pdfwf_images (full),pdfwf_compile_before_send,pdfwf_mszip,kt-ltx-chk-auto</key>
      <key name="Run after Send"></key>
      <key name="Do not copy from server"></key>
      <key name="Send to server">tex,pdf,log,aux,data,out,pgrefs,*.specials,*.pdf_print_me,*.opentype.map,*.encodings.map</key>
      <key name="Template">chencao.for</key>
      <key name="Type page width" type="float">170</key>
      <key name="Type page height" type="float">240</key>
      <key name="Text area width" type="float">122</key>
      <key name="Text area height" type="float">193</key>
      <key name="Column width" type="float">122</key>
      <key name="Run after confirm send">pdfwf_fhg,pdfwf_printpdf</key>
      <key name="Main file type">PDF</key>
      <key name="Track changes" type="boolean">No</key>
      <key name="Update pubonline date" type="boolean">No</key>
      <key name="UTF-8" type="boolean">Yes</key>
      <key name="PDF type">print</key>
      <key name="Main file name">print/&amp;ms;.pdf</key>
      <key name="MS subdirs">www,print</key>
      <key name="Do not Send subdirs">images</key>
      <key name="Update doctopic" type="boolean">No</key>
      <key name="GCNS" type="boolean">No</key>
      <key name="GCNS start date"></key>
      <key name="Prelim templates"></key>
      <key name="Differ.DVI"></key>
      <key name="Differ.DVI.Meta.Default"></key>
      <key name="Differ.DVI.Meta.Variable"></key>
    </sec>
  </sec>
</secfile>