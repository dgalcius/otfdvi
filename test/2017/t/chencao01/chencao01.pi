%Publisher ID=DGBOOK
%Project ID=CHENCAO
%MS=CHENCAO01
%Total count=1
%e-colourful count = 1
\newpi[133 337 461 504]{01f01}{{115.71mm}{58.91mm}{eps}} %e-colourful
