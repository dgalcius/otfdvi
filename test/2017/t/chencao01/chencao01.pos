\relax 
\vtexsavepos@pdf {figure1}{5594051sp}{16694611sp}{4}
