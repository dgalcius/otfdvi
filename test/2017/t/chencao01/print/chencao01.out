\BOOKMARK [0][-]{chencao01.chapter.1:1}{\376\377\0001\000\040\000I\000n\000t\000r\000o\000d\000u\000c\000t\000i\000o\000n}{}% 1
\BOOKMARK [1][-]{chencao01.section.1:2}{\376\377\0001\000.\0001\000\040\000M\000o\000t\000i\000v\000a\000t\000i\000o\000n}{chencao01.chapter.1:1}% 2
\BOOKMARK [1][-]{chencao01.section.2:3}{\376\377\0001\000.\0002\000\040\000C\000u\000r\000r\000e\000n\000t\000\040\000s\000t\000a\000t\000u\000s\000\040\000o\000f\000\040\000r\000e\000s\000e\000a\000r\000c\000h}{chencao01.chapter.1:1}% 3
\BOOKMARK [2][-]{chencao01.subsection.1:4}{\376\377\0001\000.\0002\000.\0001\000\040\000M\000o\000d\000e\000l\000i\000n\000g\000\040\000o\000f\000\040\000c\000r\000o\000w\000d\000s}{chencao01.section.2:3}% 4
\BOOKMARK [2][-]{chencao01.subsection.2:5}{\376\377\0001\000.\0002\000.\0002\000\040\000C\000o\000n\000t\000r\000o\000l\000\040\000o\000f\000\040\000c\000r\000o\000w\000d\000s}{chencao01.section.2:3}% 5
\BOOKMARK [2][-]{chencao01.subsection.3:6}{\376\377\0001\000.\0002\000.\0003\000\040\000C\000o\000m\000m\000e\000n\000t\000s}{chencao01.section.2:3}% 6
\BOOKMARK [1][-]{chencao01.section.3:7}{\376\377\0001\000.\0003\000\040\000O\000r\000g\000a\000n\000i\000z\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000t\000h\000i\000s\000\040\000b\000o\000o\000k}{chencao01.chapter.1:1}% 7
\BOOKMARK [1][-]{chencao01.section.3:9}{\376\377\000\040\000R\000e\000f\000e\000r\000e\000n\000c\000e\000s}{chencao01.chapter.1:1}% 8
